webapp/portlet/src/main/webapp/space-settings/components/SpaceSettingFormDrawer.vue

```diff
@@ -1,135 +1,143 @@
 <template>
-  <exo-drawer
```

Comment:
can you please keep the format of the file to make the PR readable, thanks.

Reply:
ok